cypress/integration/005-postcode.js

```diff
@@ -17,19 +17,57 @@ describe('Postcode page ', function () {
     cy.get('#main-content form input[type="radio"][value="yes"]').click();
     cy.get('#main-content form button.naturescot-forward-button').click();
     // ~GET `/trap-registration-number`~
+    cy.get('input').type('12345');
     // POST `/trap-registration-number`
     cy.get('#main-content form button.naturescot-forward-button').click();
     // ~GET `/postcode`~
   });
 
   it('should allow access if the user visits all the pages in order', function () {
     cy.visit('/postcode');
-    cy.get('h1').should('contain', 'Enter the postcode you used when completing your trap registration');
+    cy.get('h1').should('contain', 'What is your postcode?');
   });
 
-  it('main button should navigate to verification success', function () {
+  it('Valid input followed by continue button should navigate to postcode', function () {
     cy.visit('/postcode');
+    cy.get('input').type('G740PR');
```

Comment:
To use a real example, but to avoid annoying anyone, we normally use NatureScot HQ's postcode, `IV3 8NW`.

Reply:
Resolved in 1640fad8c32a243dffcf6c1eb680e82f1d2354f4.